src/integration-test/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitIntegrationTest.java

```diff
@@ -114,11 +114,30 @@ public void get_CalledTwice_ShouldReturnFromSnapshotInSecondTime()
     Get get = prepareGet(0, 0, NAMESPACE, TABLE_1);
 
     // Act
-    transaction.get(get);
-    transaction.get(get);
+    Optional<Result> result1 = transaction.get(get);
+    Optional<Result> result2 = transaction.get(get);
 
     // Assert
     verify(storage).get(any(Get.class));
+    assertThat(result1).isEqualTo(result2);
```

Comment:
This is not really related to this PR, but added for consistency with the next test.